numenc_module.pyi

```diff
@@ -1,25 +0,0 @@
-# stubs for the IDE to provide autocompletion.
```

Comment:
Please remove this redundant comment -- the Python users are usually familiar with stubs -- and these can be used for more than IDE autocompletion.